### ISSUE TITLE
iOS app uses too much storage space.

### ISSUE THREAD

**Issue by krille-chan** [krille-chan/fluffychat#138]
#### Description
iOS app uses too much storage space with data after awhile. See https://matrix.ross154.net/_matrix/media/r0/download/ross154.net/VqjTSmnAtDnFQcXcFpCFVYHY/img_7104.png for a screen shot/capture. Need a way to purge its caches without having to uninstalling, redownloading, reinstalling, and reconfiguring (annoying).

#### To Reproduce
Used FluffyChat a lot for weeks especially sharing medias like videos, images, and audios.

#### Additional information:
- Device: iPhone 12 mini
- OS and OS version: iOS v14.7.1
- Installed version of FluffyChat: 0.39.1 (and earlier when I noticed weeks ago)
- (Android only) Which store are you using: 
- (Android only) Are Google Services available:

**Comment by LFd3v** [comment]
Is this still an issue? I think I remember seeing something in the changelog regarding saving cache data externally so the system would take of it, and indeed FluffyChat uses now much less storage than other Matrix clients.

**Comment by antdude** [comment]
Yes, it's better since I mentioned this bug report back then. ;)

**Comment by antdude** [comment]
:(
